suite-native/module-accounts-import/src/components/AccountImportSummary.tsx

```diff
@@ -27,8 +27,11 @@ export const AccountImportSummary = ({ networkSymbol, accountInfo }: AccountImpo
     );
 
     const isAccountImportSupported = enabledNetworks.some(network => network === networkSymbol);
+    const isXpubWithSameDifferentSymbolbAlreadyImported =
```

Comment:
This is strange why do you have both `isAccountImportedAlready` and this checks?

Reply:
Fixed in new commit, that's why I put it in draft. Now it's ready for review. Selector takes in networkSymbol as an argument now and it's passed into component which is better.